lib/parse-all.js

```diff
@@ -1,5 +1,4 @@
 'use strict';
-
 const Deferred = require('./deferred');
 const parserProgressBar = require('./parser-progress-bar');
 
```

Comment:
This PR makes me realise that I think we should refactor the signature of parseAll to `parseAll(cwd, parsers)` and no longer expose all the other config settings to this method. But that's for a separate PR :)